### DISCUSSION TITLE
2021-05-26 Weekly Meeting 5 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#38] [Weekly Meetings]
Weekly Meeting 5 is happening on Wednesday 5/26/2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- Touch base on OKRs / OMI Core Values (10 min)
- Discuss setting up a monthly Retrospective for reflection, feedback, and iterative improvement (5min)
- [Chair Election Process](https://github.com/omigroup/OMI/discussions/17#discussioncomment-769909)
- [Identity Proposal](https://github.com/omigroup/OMI/issues/36)

**Comment by mrmetaverse** [comment]
Proposed from today (meet 4): Move OKR touch-base to next week. 

Also: I would also like to propose that we discuss implementing a Monthly 1-hour retrospective to establish a critical feedback loop to our organizational success: 
- What worked well for us?
- What did not work well for us?
- What actions can we take to improve our process going forward?

Any remaining time after the retro could be spent in the form of a _Lean Coffee style_ AMA/Open Discussion. In the absence of a Agile coach or Scrum Master, I would be happy to kick off the first few sessions until others feel comfortable facilitating as well.

**Comment by robertlong** [comment]
I'd like to take some time to continue some of the discussions around Identity. We're getting some feedback in [this thread](https://github.com/omigroup/OMI/issues/36) and on the [ActivityPub SocialHub](https://socialhub.activitypub.rocks/t/minimum-viable-activitypub-implementation/1790), I think it's time to touch base and see how people are feeling. We can open the floor for anyone to discuss additional proposals on identity or ask questions / pose concerns on any of the solutions proposed so far.

**Reply by robertlong** [reply]
We also need to discuss the chair election process. https://github.com/omigroup/OMI/discussions/17#discussioncomment-769909

**Comment by robertlong** [comment]
Unfortunately, my recording for this meeting didn't correctly capture the audio. Here are some of my notes:

Identity:
- Fields in a person's profile should be able to be hidden unless made public. There was a discussion on whether or not visibility to friends is a good enough model for privacy.
- Permission prompts that detail what profile fields are being collected were discussed as a good way of permitting using user data.
- We discussed the possibility of avatars being represented as inventory items. Presumably the concepts of ownership, trading, and browsing them would be inherited from the inventory spec.
- Permissions for viewing a certain avatar/item could be represented in the inventory spec, making sure that you can tag an avatar/item as private or not.
- Avatars are often represented as multiple pieces, combined to form a single avatar. This could include different body styles, hair, clothing, and accessories. This could be stored as multiple glTF models and combined into a single one by engines. However, this is a pretty complex place to start and a single glTF/VRM model seems more achievable.
- We need the ability to have multiple identities. People represent themselves differently in different contexts, they may also may identity as [plural](https://pluralityresource.org/plurality-information/) and may front as multiple identities.
- It is incredibly important that identities cannot be publicly associated with each other if people wish for them to remain private. Protecting against fingerprinting is a difficult problem in XR where gait detection can accurately identify someone. While gait detection may be out of scope for an identity spec, we do need to keep fingerprinting and privacy in mind.
